src/devices/src/virtio/net/event_handler.rs

```diff
@@ -12,82 +12,34 @@ use crate::virtio::{VirtioDevice, RX_INDEX, TX_INDEX};
 
 impl Net {
     fn process_activate_event(&self, event_manager: &mut EventManager) {
+        debug!("net: activate event");
+        if let Err(e) = self.activate_evt.read() {
+            error!("Failed to consume net activate event: {:?}", e);
```

Comment:
Do we need to bail out and never activate this device ?

Reply:
The device is already activated at this point, I think this falls under the category of: log error and carry on. These are highly unlikely errors which used to be `unwrap()`s before and we switched them to logging to make sure we don't crash the whole guest on a device error.

The perfect solution would probably be to reset the device on such errors, but we don't have that support yet.